moveit_plugins/moveit_simple_controller_manager/src/follow_joint_trajectory_controller_handle.cpp

```diff
@@ -97,6 +98,19 @@ bool FollowJointTrajectoryControllerHandle::sendTrajectory(const moveit_msgs::ms
   return true;
 }
 
+void FollowJointTrajectoryControllerHandle::setPathTolerance(double /*path_tolerance*/)
+{
```

Comment:
Let's print a warning that they're not doing anything at the moment. Why do we need them in the first place?